app/components/dialog/components/modifyProfile/index.tsx

```diff
@@ -113,22 +113,21 @@ class ModifyProfileDialog extends React.PureComponent<
                   name="currentAffiliation"
                   component={AffiliationSelectBox}
                   inputClassName={styles.inputField}
-                  placeholder="Current Affiliation"
                   format={this.formatAffiliation}
                 />
               </div>
             </div>
             <div className={styles.bioWrapper}>
               <label htmlFor="bio">
-                Short Bio<small> (optional)</small>
+                Short Bio<small> (Optional)</small>
               </label>
               <Field
                 name="bio"
                 component={ReduxAutoSizeTextarea}
                 type="text"
                 textareaClassName={styles.textAreaWrapper}
                 textareaStyle={{ padding: "8px" }}
-                placeholder="Please tell us about yourself."
+                placeHolder="Please tell us about yourself."
```

Comment:
`placeholder` is the right attribute.

    interface InputHTMLAttributes<T> extends HTMLAttributes<T> {
        accept?: string;
        alt?: string;
        autoComplete?: string;
        autoFocus?: boolean;
        capture?: boolean | string; // https://www.w3.org/TR/html-media-capture/#the-capture-attribute
        checked?: boolean;
        crossOrigin?: string;
        disabled?: boolean;
        form?: string;
        formAction?: string;
        formEncType?: string;
        formMethod?: string;
        formNoValidate?: boolean;
        formTarget?: string;
        height?: number | string;
        list?: string;
        max?: number | string;
        maxLength?: number;
        min?: number | string;
        minLength?: number;
        multiple?: boolean;
        name?: string;
        pattern?: string;
        **placeholder?: string;**
        readOnly?: boolean;
        required?: boolean;
        size?: number;
        src?: string;
        step?: number | string;
        type?: string;
        value?: string | string[] | number;
        width?: number | string;

        onChange?: ChangeEventHandler<T>;
    }

Reply:
The name of the placeholder was defined as placeHolder in the props interface of ReactAutoSizeTextarea.

~~~
interface ReduxAutoSizeTextareaProps extends WrappedFieldProps {
  wrapperClassName?: string;
  wrapperStyle?: React.CSSProperties;
  textareaStyle?: React.CSSProperties;
  textareaClassName?: string;
  placeHolder?: string;
  rows?: number;
  disabled: boolean;
}
~~~

 But I agree that we should unify the spelling. I'll make that change.

Reply:
👍 👍 👍